Source/Tests/TrackableEntities.Client.Tests/ChangeTrackingCollectionTests.cs

```diff
@@ -719,6 +719,23 @@ public void EntityChanged_Event_Should_Fire_When_Items_Added_Modified_Deleted()
             Assert.Equal(3, changesCount);
         }
 
+        [Fact]
+        public void GraphChanged_Event_Should_Fire_When_Graph_Items_Added_Modified_Deleted()
+        {
+          //Arrange
+          var database = new MockNorthwind();
+          var product = database.Products[0];
+          var changeTracker = new ChangeTrackingCollection<Product>(product);
+          var raised = false;
+          changeTracker.GraphChanged += (s, e) => raised = true;
```

Comment:
What do you mean? Should we keep only one event that fires when the graph is changed?